<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true">background-color:white;</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="styleSheet">
              <string notr="true">border:none;</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_11">
                <property name="styleSheet">
                 <string notr="true">font-size:18px</string>
                </property>
                <property name="text">
                 <string>Search For A Cardet</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignVCenter">
               <widget class="QGroupBox" name="groupBox_3">
                <property name="styleSheet">
                 <string notr="true">border:none</string>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLineEdit" name="lineEdit_2">
                   <property name="styleSheet">
                    <string notr="true">min-width: 350px; min-height:40px;border-radius:10px;border:1px solid grey;padding-left:20px;padding-right:10px;font-size:15px;color:grey;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_3">
                   <property name="styleSheet">
                    <string notr="true">min-width:100px;background-color:#1AAA62; min-height:40px;color:white;border:none;border-radius:10px;</string>
                   </property>
                   <property name="text">
                    <string>Search</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>30</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QPushButton" name="AddCrimeButton">
                <property name="styleSheet">
                 <string notr="true">background-color:white; min-height:40;min-width:150;border:1px solid grey;border-radius:10px;</string>
                </property>
                <property name="text">
                 <string>Add a Crime</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="styleSheet">
           <string notr="true">background-color:white;</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QGroupBox" name="groupBox_6">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>75</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border:nne</string>
             </property>
             <property name="title">
              <string>GroupBox</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="sizeConstraint">
               <enum>QLayout::SetMaximumSize</enum>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>9</number>
              </property>
              <item>
               <widget class="QLineEdit" name="lineEdit_3">
                <property name="styleSheet">
                 <string notr="true">min-height:40;min-width:300;border-radius:10px;border:1px solid grey;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_5">
                <property name="styleSheet">
                 <string notr="true">min-height:40;min-width:100;background-color:#1AAA62;color:white;border:none; border-radius:10</string>
                </property>
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_6">
                <property name="styleSheet">
                 <string notr="true">min-height:40px;min-width:100;border:1px solid grey; border-radius:10</string>
                </property>
                <property name="text">
                 <string>Add A Crime</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">padding:0;border:none;</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QGroupBox" name="groupBox_7">
                <property name="title">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_8">
                <property name="styleSheet">
                 <string notr="true">padding:10px;</string>
                </property>
                <property name="title">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox_9">
          <property name="styleSheet">
           <string notr="true">background-color:white;</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QGroupBox" name="groupBox_12">
             <property name="styleSheet">
              <string notr="true">border:none</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">font-size:18px; font-weight:500;</string>
                </property>
                <property name="text">
                 <string>Add A New Crime</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_13">
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QLineEdit" name="lineEdit_4">
                   <property name="styleSheet">
                    <string notr="true">min-width:300;min-height:40;border-radius:20px;border:1px solid grey; padding-left:20px; padding-right:20px;font-size:15px;color:grey;</string>
                   </property>
                   <property name="text">
                    <string>NDA Number</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_4">
                   <property name="styleSheet">
                    <string notr="true">min-height:40;background-color:white;border-radius:20px;border:1px solid grey;color:#1AAA62;font-weight:600;font-size:15px;min-width:80;</string>
                   </property>
                   <property name="text">
                    <string>GO</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item alignment="Qt::AlignHCenter">
         <widget class="QGroupBox" name="groupBox_10">
          <property name="styleSheet">
           <string notr="true">border:none;</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">font-weight:500;font-size:18px</string>
             </property>
             <property name="text">
              <string>Add A New Crime</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fullname">
             <property name="styleSheet">
              <string notr="true">border:1px solid grey;min-width:350;min-height:40;border-radius:20;padding-left:20px;padd-right:20px;color:grey;</string>
             </property>
             <property name="text">
              <string>Full Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ndanumber">
             <property name="styleSheet">
              <string notr="true">border:1px solid grey;min-width:350;min-height:40;border-radius:20;padding-left:20px;padd-right:20px;color:grey;</string>
             </property>
             <property name="text">
              <string>NDA Number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="department">
             <property name="styleSheet">
              <string notr="true">border:1px solid grey;min-width:350;min-height:40;border-radius:20;padding-left:20px;padd-right:20px;color:grey;</string>
             </property>
             <property name="text">
              <string>Department</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="state">
             <property name="styleSheet">
              <string notr="true">border:1px solid grey;min-width:350;min-height:40;border-radius:20;padding-left:20px;padd-right:20px;color:grey;</string>
             </property>
             <property name="text">
              <string>State</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dateofbirth">
             <property name="styleSheet">
              <string notr="true">border:1px solid grey;min-width:350;min-height:40;border-radius:20;padding-left:20px;padd-right:20px;color:grey;</string>
             </property>
             <property name="text">
              <string>Date Of Birth</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sex">
             <property name="styleSheet">
              <string notr="true">border:1px solid grey;min-width:350;min-height:40;border-radius:20;padding-left:20px;padd-right:20px;color:grey;</string>
             </property>
             <property name="text">
              <string>Sex</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="battalion">
             <property name="styleSheet">
              <string notr="true">border:1px solid grey;min-width:350;min-height:40;border-radius:20;padding-left:20px;padd-right:20px;color:grey;</string>
             </property>
             <property name="text">
              <string>Battalion</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="description">
             <property name="styleSheet">
              <string notr="true">border:1px solid grey;min-width:350;min-height:40;border-radius:20;padding-left:20px;padd-right:20px;color:grey;</string>
             </property>
             <property name="text">
              <string>Crime Description</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="imageBotton">
             <property name="styleSheet">
              <string notr="true">min-width:350;min-height:40;background-color:grey;color:white;border-radius:15</string>
             </property>
             <property name="text">
              <string>Add Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="styleSheet">
              <string notr="true">height:40;min-width:350;background-color:#1AAA62;color:white;border-radius:15px</string>
             </property>
             <property name="text">
              <string>Submit</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QGroupBox" name="groupBox_11">
          <property name="styleSheet">
           <string notr="true">border:none;</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">font-weight:500;font-size:18px</string>
             </property>
             <property name="text">
              <string>Add A New Crime</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">font-size:16px;font-weight:500;</string>
             </property>
             <property name="text">
              <string>John Doe</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit">
             <property name="styleSheet">
              <string notr="true">border:1px solid grey; border-radius: 20px;padding:20px;color:grey;min-width:300px;</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QPushButton" name="pushButton_8">
             <property name="styleSheet">
              <string notr="true">min-height:40px;background-color:#1AAA62;max-width:200;color:white;border-radius:15;min-width:200px;</string>
             </property>
             <property name="text">
              <string>Submit</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
